### ISSUE TITLE
question property startWithNewLine is not kept when deleting previous question

### ISSUE THREAD

**Issue by hekin1** [surveyjs/survey-creator#3364]
### Are you requesting a feature, reporting a bug or ask a question?
reporting a bug

### What is the current behavior?
When a question is deleted, the layout is not computed against the remaining questions `startWithNewLine` property.
Example: Given a survey with 3 questions where question3 has `startWithNewLine=false` :

![Screenshot_122](https://user-images.githubusercontent.com/14329635/185415974-c3e47f3b-4657-4195-b824-0995371fde75.png)

If user deletes question2 it is expected that question3 still keeps the property layout of not beginning with a new line, but instead, question3 is moved to a new line.

### What is the expected behavior?
When deleting questions, remaining questions should keep their layout regards to not beginning on new lines.

### How would you reproduce the current behavior (if this is a bug)?
See this plunker: https://plnkr.co/edit/nbhxXpQMfw5b5Kdt
Delete question2 and observe that question3 gets added to a new line, even if it has `startWithNewLine=false`

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:
https://plnkr.co/edit/nbhxXpQMfw5b5Kdt

Test code

```js
{
     "type": "text",
     "name": "question3",
     "startWithNewLine": false
 }

```

### Specify your

* browser: firefox and chrome
* editor version: 1.9.45

**Comment by andrewtelnov** [comment]
@hekin1 We will look.

Thank you,
Andrew

**Comment by andrewtelnov** [comment]
@hekin1 I have fixed it by setting startWithNewLine to `true` for the new first element. I believe the most users will expect this behavior.
Here is [the related issue](https://github.com/surveyjs/survey-library/issues/4735) in library.

Thank you,
Andrew

**Comment by hekin1** [comment]
@andrewtelnov 
Thanks for giving this issue attention! 
I though I would leave som feedback from our users for the solution you provided in 1.9.46.
Our users thought it was confusing that questions they actively marked as (unchecked = should NOT start on new line) suddenly got checked automatically to (checked = should start on new line). Questioning them made it clear they were expecting the questions to keep its property (unchecked = should start on new line), when question before was deleted.
So in other words following example. Question C in the example below user has unchecked start on new line = false
```
A
B C
```
They were expecting that when deleting question B, question C would keep its configuration:
```
A C
```
But instead it works like this after your fix:
```
A
C
```

Anyways, I just though I would drop the feedback. 
Thanks again and best regards,

**Comment by andrewtelnov** [comment]
@hekin1 I can agree that it may be the case. However, it is not always the case. Let's consider the following scenario:
A B 
C D
or 
A B 
C D E

In this case I would prefer to have:
A B
C
and
A B
D E
instead of:
A  B C
and 
A B D E

Thank you,
Andrew
